eventuate-adapter-spark/src/main/scala/com/rbmhtechnology/eventuate/adapter/spark/SparkStreamAdapter.scala

```diff
@@ -235,4 +235,3 @@ private class DurableEventReceiver(id: String, connection: ReplicationConnection
       readSchedule.foreach(_.cancel())
   }
 }
-
```

Comment:
Please undo all changes to files that are not related to the features of this PR.